tests/test_file_upload.py

```diff
@@ -31,3 +31,37 @@ def begin_fn(interface):
     files = [dict(files_dict=dict(files=[("test.txt", "now")]))]
     ctx_util = publish_util(files=files, begin_cb=begin_fn)
     assert "test.txt" in ctx_util.file_names
+
+
+def test_file_upload_azure_good(mocked_run, publish_util, mock_server):
+    mock_server.set_context("emulate_azure", True)
+
+    def begin_fn(interface):
+        # We use the sketchy hack of naming our file after azure blob storage
+        # to trigger the azure logic
```

Comment:
I don't understand what this comment refers to, could you elaborate please?

Reply:
Ahh, I was doing something sketchier initially.  I got rid of the sketch, will remove the comment.